.travis.yml

```diff
@@ -1,23 +1,24 @@
+---
 language: ruby
 sudo: false
 rvm:
   - 1.9.3
-  - 2.0.0
-  - 2.1.10
-  - 2.2.9
-  - 2.3.6
-  - 2.4.3
-  - 2.5.0
-  - rbx-2
+  - 2.0
+  - 2.1
+  - 2.2
+  - 2.3
+  - 2.4
+  - 2.5.1
```

Comment:
Why is this different from all the other version definitions?

Reply:
Fixed in d548ee427bc8e6e308f224897231823f0e86fa04